exercises/Ex7CheckInt.js

```diff
@@ -0,0 +1,9 @@
+function checkInt(num){
+    if((num >= 20 && num <=100) || num == 400 ){
+        console.log("true");
+    }
+    else{
+        console.log("false");
+    }
+}
```

Comment:
As discussed: avoid output in methods which are intended to just return true/false.